### ISSUE TITLE
Date Picker error

### ISSUE THREAD

**Issue by kongming0gn** [ButchersBoy/MaterialDesignInXamlToolkit#403]
Hi there.. Please help me, I'm getting the following error whenever I try to click on DatePicker:

"An unhandled exception of type 'System.Windows.Markup.XamlParseException' occurred in PresentationFramework.dll

Additional information: The sum of AccelerationRatio and DecelerationRatio must be less than or equal to one."

I have both mahapps and materialdesign installed. thanks in advance!

**Comment by ButchersBoy** [comment]
What version are you using?

**Comment by kongming0gn** [comment]
It's as follows:

MaterialDesignThemes - 1.5.0.523
MaterialDesignColors - 1.1.2
MaterialDesignThemes.MahApps - 0.0.4
MahApps.Metro - 1.2.4

**Comment by kongming0gn** [comment]
By the way, even the calendar control is giving me the same exception. But materialDesign:TimePicker is working fine.

**Comment by ButchersBoy** [comment]
OK, can you try the calendar on 2.0 (should already be fixed).  I will look at the TimePicker seperately.

**Comment by kongming0gn** [comment]
sorry, i'm a bit new at this.. but should i uninstall it first, and is it only the materialdesignthemes or all of it? thanks...

**Comment by ButchersBoy** [comment]
Are you using Nuget to install?  2.0 is currently in pre-release.  You can install pre-release nuget packages from the nuget console, or from the nuget gui by changing the search options.

**Comment by kongming0gn** [comment]
Thanks, i installed the latest update dated today. But It got me a couple on the error list...

- Type reference cannot find type named '{clr-namespace:MaterialDesignThemes.Wpf;assembly=MaterialDesignThemes.Wpf}HintAssist'.
- The property 'TextFieldAssist.Hint' does not exist in XML namespace 'http://materialdesigninxaml.net/winfx/xaml/themes'.
- The member "Hint" is not recognized or is not accessible.
- The attachable property 'Hint' was not found in type 'TextFieldAssist'.
- Cannot find resource named 'FlyoutHeaderFontSize'. Resource names are case sensitive.

**Comment by ButchersBoy** [comment]
Please see https://github.com/ButchersBoy/MaterialDesignInXamlToolkit/wiki/Version-2.0.0---Breaking-Changes

**Comment by kongming0gn** [comment]
Ok, thanks a lot! It's working now!

**Comment by PushyHedge** [comment]
I have got he latest version Nuget and have put this in my XMAL
xmlns:materialDesign="http://materialdesigninxaml.net/winfx/xaml/themes"

 <TextBox  materialDesign:HintAssist.Hint="hello world" 
                                      materialDesign:HintAssist.IsFloating="False"/>

But I get the compile errors:
The name "HintAssist" does not exist in the namespace "http://materialdesigninxaml.net/winfx/xaml/themes".
The attachable property 'IsFloating' was not found in type 'HintAssist'.

I looked at your link above, but it doesn;t make any sense to me (just telle me it's moved to HintAssist.
